apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -35,6 +35,7 @@ type MultiClusterAppSpec struct {
 	Roles                []string        `json:"roles,omitempty" norman:"type=array[reference[roleTemplate]],required"`
 	RevisionHistoryLimit int             `json:"revisionHistoryLimit,omitempty" norman:"default=10"`
 	UpgradeStrategy      UpgradeStrategy `json:"upgradeStrategy,omitempty"`
+	HelmVersion          string          `json:"helmVersion,omitempty" norman:"nocreate,noupdate"`
```

Comment:
Let's move this to status instead. Spec indicates something that a user sets while status is controlled internally. Also in the future status will be protected similar to how k8s does it but through 'steve'.